config/jenkins_infra.ci.jenkins.io.yaml

```
@@ -292,11 +292,14 @@ controller:
                     EOF
                     systemctl daemon-reload
                     systemctl restart docker
+                    export DEFAULT_JDK=/opt/jdk-11
+                    update-alternatives --install /usr/bin/java java "^${DEFAULT_JDK}/bin/java" 2000
+                    echo "JAVA_HOME=^${DEFAULT_JDK}" >> /etc/environment
                     rm -f /etc/sudoers.d/90-cloud-init-users
                     echo "END CLOUDINIT"
                   javaPath: "/opt/jdk-17/bin/java"
                   jvmOptions: "-XX:+PrintCommandLineFlags"
-                  labels: "linux-amd64 linux-amd64-docker azure-vms-jenkins-sponsorship"
+                  labels: "linux-amd64 linux-amd64-docker azure-vms-jenkins-sponsorship jdk11 maven-11"
```

```suggestion
                  labels: "linux-amd64 linux-amd64-docker azure-vms-jenkins-sponsorship"
```

We discussed about using the ARM64 as default in this controller (we like to live on the edge ;) )